### DISCUSSION TITLE
Email not using configuration settings?

### DISCUSSION THREAD

**Discussion by jmourinho** [Leantime/leantime#959] [General]
No matter what i set at the configuration.php file at 

`public $email = 'whateveriset@domain.com'`

Emails are being always sent from _noreply@mysubdomain.domain.com_ (subdomain and domain named, of course), while they should be sent from _whateveriset@domain.com_ as defined.

I am using SMTP, not phpmail functions. Am i forgetting something?

Thkx

**Comment by phulstaert** [comment]
Hi @jmourinho. The settings in the configuration.php are overruled if an environment variable is set.
This should be the `LEAN_EMAIL_RETURN` variable.
If you are using docker-compose, you can find it in the docker-compose.yml file.
If you are using docker, it is found in your docker command after the -e parameter.    (-e LEAN_EMAIL_RETURN=noreply@.....)

Another possibility is that this is something at the mail server level.
Have you tried connecting with a mail client and sending a test mail? Does this work?

**Reply by jmourinho** [reply]
Hi @phulstaert thank you for your answer.

I understand, but the LEAN_EMAIL_RETURN variable is assigned to the $email value as this line:

`$this->email = $this->configEnvironmentHelper("LEAN_EMAIL_RETURN", $this->email);`

As i have `public $email = 'innovation@mycompany.com'`, emails should be being sent from this email and not from no-reply@mycompany.com"

I am using an external SMTP server to send emails, but it is no problem as i am using it to send emails from a set of wordpress sites with no problem.

**Reply by phulstaert** [reply]
You first declare the $email variable with a value.
Next, we call a function (configEnvironmentHelper) that checks if a LEAN_EMAIL_RETURN variable is set. If it is set, it will overwrite the $email variable; if not, it will keep the $email value.

If you have set LEAN_EMAIL_RETURN, it does not matter what you set in the configuration.php file.

**Reply by jmourinho** [reply]
Thank you. So where can i set the LEAN_EMAIL_RETURN? (i am not using docker)

**Reply by phulstaert** [reply]
If you are not using docker or docker-compose, then there probably is no LEAN_EMAIL_RETURN environment variable defined.
You could always check by running `env` in the terminal. (you are using a Linux distribution?)
If too much info is shown, then you can run `env |grep "LEAN_"` to limit the variables.
If it is not in there, something else is going on.

Also check your login credentials, that you use the ones related with innovation@... and not the ones related to no-reply@....

If this is correct, I would recommend checking the logging from the SMTP server. This server can determine what was sent to it.
